afpacket/afpacket.go

```diff
@@ -39,6 +39,12 @@ import "C"
 var pageSize = int(C.getpagesize())
 var tpacketAlignment = uint(C.TPACKET_ALIGNMENT)
 
+// ErrPoll returns by poll
+var ErrPoll = errors.New("Error condition")
```

Comment:
Could you make this error message more descriptive?  Something like "packet poll failed"

Also, please make error messages start with lower-case characters.